modules/tide_content_collection/src/Plugin/Field/FieldWidget/ContentCollectionConfigurationWidget.php

```diff
@@ -0,0 +1,873 @@
+        '#default_value' => $settings['tabs']['content']['content_types_wrapper']['contentTypes'] ?? [],
+        '#weight' => 1,
+      ];
+    }
+
+    if ($this->indexHelper->isFieldTopicIndexed($index)) {
+      $element['tabs']['content']['field_topic_wrapper'] = [
+        '#type' => 'details',
+        '#title' => 'Field Topic',
+        '#open' => FALSE,
+        '#collapsible' => TRUE,
+        '#weight' => 2,
+      ];
+      $element['tabs']['content']['field_topic_wrapper']['enable_field_topic'] = [
+        '#type' => 'checkbox',
+        '#title' => $this->t('Enable field topic'),
+        '#default_value' => $settings['tabs']['content']['field_topic_wrapper']['enable_field_topic'] ?? FALSE,
+      ];
+      $element['tabs']['content']['field_topic_wrapper']['show_field_topic_operator'] = [
+        '#type' => 'checkbox',
+        '#title' => $this->t('Show field topic operator'),
+        '#default_value' => $settings['tabs']['content']['field_topic_wrapper']['show_field_topic_operator'] ?? FALSE,
+      ];
+    }
+
+    if ($this->indexHelper->isFieldTopicIndexed($index)) {
+      $element['tabs']['content']['field_tags_wrapper'] = [
+        '#type' => 'details',
+        '#title' => 'Field Tags',
+        '#open' => FALSE,
+        '#collapsible' => TRUE,
+        '#weight' => 2,
+      ];
+      $element['tabs']['content']['field_tags_wrapper']['enable_field_tags'] = [
+        '#type' => 'checkbox',
+        '#title' => $this->t('Enable field tags'),
+        '#default_value' => $settings['tabs']['content']['field_tags_wrapper']['enable_field_tags'] ?? FALSE,
+      ];
+      $element['tabs']['content']['field_tags_wrapper']['show_field_tags_operator'] = [
+        '#type' => 'checkbox',
+        '#title' => $this->t('Show field tags operator'),
+        '#default_value' => $settings['tabs']['content']['field_tags_wrapper']['show_field_tags_operator'] ?? FALSE,
+      ];
+    }
+
+    $entity_reference_fields = $this->getEntityReferenceFields();
+    if (!empty($entity_reference_fields)) {
+      foreach ($entity_reference_fields as $field_id => $field_label) {
+        $element['tabs']['content'][$field_id . '_wrapper'] = [
+          '#type' => 'details',
+          '#title' => $field_label,
+          '#open' => FALSE,
+          '#collapsible' => TRUE,
+          '#weight' => 2,
+        ];
+        $element['tabs']['content'][$field_id . '_wrapper']['enable_' . $field_id] = [
+          '#type' => 'checkbox',
+          '#title' => $this->t('Enable @field_id', ['@field_id' => str_replace('_', ' ', $field_id)]),
+          '#default_value' => $settings['tabs']['content'][$field_id . '_wrapper']['enable_' . $field_id] ?? FALSE,
+        ];
+        $element['tabs']['content'][$field_id . '_wrapper']['show_' . $field_id . '_operator'] = [
+          '#type' => 'checkbox',
+          '#title' => $this->t('Show @field_id operator', ['@field_id' => str_replace('_', ' ', $field_id)]),
```

Comment:
just `Show filter operator`. It does not make sense when outputting something like `Show field p term reference operator` etc.

Reply:
Updated to correct labels.